docs/notebooks/sampling_positions.ipynb

```diff
@@ -0,0 +1,203 @@
+    "(ra, dec) = uniform_sampler.generate(num_samples=500)\n",
+    "\n",
+    "hp.mollview(hold=True)\n",
+    "hp.visufunc.projscatter(ra, dec, lonlat=True)\n",
+    "hp.graticule()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However this approach has limited use when simulating a specific survey. Depending on the survey's coverage, a significant number of (RA, dec) points may fall outside the viewing area.\n",
+    "\n",
+    "## Sampling from a Survey\n",
+    "\n",
+    "We can sample (RA, dec) coordinates from a survey (an `OpSim` object) in two ways. First we could sample a pointing from the survey and then a point from that field of view. Second, we could sample uniformly from the region coverage by the survey.\n",
+    "\n",
+    "We consider each of these approaches below.\n",
+    "\n",
+    "### Sampling Pointings\n",
+    "\n",
+    "Sampling pointings from the survey provides a visit-weighted sampling of positions covered by the survey. For concreteness let's start with a survey that visits two fields: one centered at (45.0, -15.0) and the other at (315.0, 15.0). The first field is visited once and the second field is visited four times on four consecutive nights."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "values = {\n",
+    "    \"observationStartMJD\": np.array([0.0, 1.0, 2.0, 3.0, 4.0]),\n",
+    "    \"fieldRA\": np.array([45.0, 315.0, 315.0, 315.0, 315.0]),\n",
+    "    \"fieldDec\": np.array([-15.0, 15.0, 15.0, 15.0, 15.0]),\n",
+    "    \"zp_nJy\": np.ones(5),\n",
+    "}\n",
+    "opsim = OpSim(values, radius=30.0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can sample from these pointings using the `OpSimRADECSampler` node."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pointing_sampler = OpSimRADECSampler(opsim, radius=30.0, node_label=\"opsim\")\n",
+    "(ra, dec, time) = pointing_sampler.generate(num_samples=100)\n",
+    "\n",
+    "hp.mollview(hold=True)\n",
+    "hp.visufunc.projscatter(ra, dec, lonlat=True)\n",
+    "hp.graticule()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, the field centered in the Northern hemisphere is sampled significantly more than the one centered in the Southern hemisphere.\n",
+    "\n",
+    "### Sampling Coverage\n",
+    "\n",
+    "If we instead would like to sample uniformly from the area covered by the survey, we have two options. The `OpSimUniformRADECSampler` uses rejection sampling to generate positions. This approach can be slow for surveys with small coverage. However, since it requires no pre-computation, it is a viable option for generating a small number samples from a survey with high coverage."
```

Comment:
```suggestion
    "If we instead would like to sample uniformly from the area covered by the survey, we have two options. The `OpSimUniformRADECSampler` uses rejection sampling to generate positions. This approach can be slow for surveys with small coverage. However, since it requires no pre-computation, it is a viable option for generating a small number of samples from a survey with high coverage."
```